src/components/ChatWindow/ChatWindow.tsx

```diff
@@ -39,17 +40,26 @@ const ChatWindow = ({ onChatActivation }: Props) => {
 
     // Send the message to the server
     try {
+      setIsLoading(true)
       const response = await ChatAPI.send({
         botId: BOT_ID,
         chatSessionId: CHAT_SESSION_ID,
         command: message,
       })
 
       // Handle the response from the server and update chat history
-      console.log("response", response)
       addMessageToChatHistory(response.output, "bot")
-    } catch (error) {
+    } catch (error: unknown) {
       console.error("Error sending message:", error)
+
+      // If else statement ensures that error is an instance of Error before accessing its properties. If it's not, it adds a generic error message to the chat history.
+      if (error instanceof Error) {
+        addMessageToChatHistory(error.message, "bot")
+      } else {
+        addMessageToChatHistory("An unknown error occurred", "bot")
+      }
```

Comment:
just `setError(true)` and show a generic error message on the UI.

Reply:
I just update on my newest commit. Add `Alert` component to display message when there is error. 
<img width="306" alt="image" src="https://github.com/AI-ChatBot-Yoga/ai-chatbot/assets/133303660/7f1735b3-63af-48fe-9b71-e65335bf9113">